core/shared/src/main/scala/spire/math/Numeric.scala

```diff
@@ -229,7 +229,18 @@ private[math] class RealIsNumeric extends Numeric[Real] with RealIsFractional wi
   override def toDouble(n: Real): Double = n.toDouble
 }
 
+/* TODO: what to do with complex, which is not ordered?
```

Comment:
This is probably a good question for @non  - but given that `Numeric` just isn't used much, I'm OK with losing `Numeric[Complex[A]]`.

The alternative is that we just stub out the ordering or do something janky. `Numeric` basically exists as a "best" effort type class and it should be somewhat accepted that not all operations may exist/obey useful laws... Just typing that makes me think `Numeric` should just kind of disappear...